WordPress/src/main/java/org/wordpress/android/ui/history/HistoryDetailContainerFragment.java

```diff
@@ -172,6 +179,37 @@ public void onClick(View view) {
         return rootView;
     }
 
+    private void mapRevisions() {
+        if (getArguments() != null) {
+            mRevision = getArguments().getParcelable(EXTRA_CURRENT_REVISION);
+
+            final long[] previousRevisionsIds = getArguments().getLongArray(EXTRA_PREVIOUS_REVISIONS_IDS);
+            final List<RevisionModel> revisionModels = new ArrayList<>();
+            for (final long revisionId : previousRevisionsIds) {
+                revisionModels.add(mPostStore.getRevisionById(revisionId));
```

Comment:
This is probably not a significant issue, but I wonder if performance might suffer when accessing DB from main thread for a post wit a lot of changes.